config.yaml

```diff
@@ -10,9 +10,4 @@ options:
       Acceptable values are: "info", "debug", "warning", "error" and "critical"
     default: "info"
     type: string
-  kratos_url:
```

Comment:
I think our PRs in this repo will be in conflict - I opened one for the kratos integration. I'm not sure if we want to add all relations in this PR or split them into different ones. @massigori what do you think?